README.md

```diff
@@ -117,6 +117,7 @@ Customize the integration environment variables configurations with the paramete
 | LOGZIO_RETRY_MAX_INTERVAL | **Default**: `30s` <br>  Maximum interval, in seconds, to wait between retries. |
 | LOGZIO_FLUSH_THREAD_COUNT | **Default**: `2` <br>  Number of threads to flush the buffer. |
 | LOGZIO_LOG_LEVEL | **Default**: `info` <br> The log level for this container. |
+| INCLUDE_NAMESPACE | **Default**: `""` <br> Use if you wish to send logs from specific k8s namespaces. Should be in the following format: <br> `kubernetes.var.log.containers.**_<<NAMESPACE-TO-INCLUDE>>_** kubernetes.var.log.containers.**_<<ANOTHER-NAMESPACE>>_**`. |
```

Comment:
Not very clear, what happens if you don't fill this parameter? will it take from all?
```suggestion
| INCLUDE_NAMESPACE | **Default**: `""` <br> Use if you wish to send logs from specific k8s namespaces, space delimited. Should be in the following format: <br> `kubernetes.var.log.containers.**_<<NAMESPACE-TO-INCLUDE>>_** kubernetes.var.log.containers.**_<<ANOTHER-NAMESPACE>>_**`. |
```